modules/admin/proc-rbac-config-conditional-policy-file.adoc

```diff
@@ -0,0 +1,118 @@
+[id='proc-rbac-config-conditional-policy-file_{context}']
+= Configuring conditional policies defined in an external file
+
+You can configure and manage conditional policies that are defined in an external file.
```

Comment:
Consider adding something about why customers would want to do that. Something like:

To define conditional policies, rather than using {product} web UI or API, you might want to edit directly the configuration files, and pass them to {product-short}.
You can upload the files to {ocp} and configure {product-short} to use these files rather than the default files.

(I am not sure about the file location, though)

Reply:
@PatAKnight @AndrienkoAleksandr Could you please review the following introduction? We're planning to add this to the section:

_To define conditional policies, you can directly edit the configuration files and pass them to Developer Hub, instead of using the Developer Hub web UI or API. You can upload these files to OCP, and configure Developer Hub to use these files instead of the default files._